### PR TITLE
Add support for MySQL StartTLS in connect

### PR DESCRIPTION
**Pull request by csstaub**
Use `certigo connect --start-tls=mysql HOST:PORT` to dump certs from a MySQL server.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -150,6 +147,33 @@ func main() {
 	}
 }
 
+func getConnectionState() *tls.ConnectionState {
```
Comment (mcpherrinm) [inline main.go]: I'd prefer if this function took arguments instead of relying on the globals connectStartTLS and connectTo

Comment (csstaub) [inline main.go]: tlsConfigForConnect() also depends on global state though, so what's the point?

**Comment by csstaub** [comment]
@mcpherrinm Split it out into multiple commits.

File: main.go (hunk 1 above)
Comment (mcpherrinm) [inline main.go]: ok

**Pull request merged**